app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_user!
+
+  def index
+    @users = policy_scope(User)
+    authorize User
+    @roles = Role.all
+    @current_user = current_user
+  end
+
+  def update
+    @user = User.find(params[:id])
+    authorize @user
+    @user.update(secure_params)
+    render users_url
+  end
+
+  def destroy
+    @users = User.all
```

Comment:
why do have ` @users = User.all` here and ` @users = policy_scope(User)` in index???

are they different somehow? it's good to have similar code

Reply:
Will be fixed